warpstream/metadata.csv

```diff
@@ -0,0 +1,137 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+warpstream.can_connect,gauge,,,,"1 if can connect to Warpstream Agent, otherwise 0",0,warpstream,can_connect,
+warpstream.agent_connection_throttle_count,count,,,,"number of new connections opened to the Warpstream Agent throttled",1,warpstream,agent_connection_throttle_count,
+warpstream.agent_deadscanner_optimistic_queue_delete_file_outcome,count,count,,,"incremented whenever the Warpstream Agent processed a file in its deadscanner optimistic queue, tagged by 'outcome'",1,warpstream,agent_deadscanner_optimistic_queue_delete_file_outcome,
+warpstream.agent_deadscanner_optimistic_queue_submit_outcome,count,,,,"incremented whenever we submitted a request to the deadscanner optimistic queue, tagged by 'outcome'",1,warpstream,agent_deadscanner_optimistic_queue_submit_outcome,
```

Comment:
I recommend avoiding "we", since the user may not know who you mean. Who is submitting the request? Note that there are a few other instances of "we" in this file, I've just flagged one.

Reply:
thanks!